vocab/context.jsonld

```diff
@@ -0,0 +1,87 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "cred": "https://w3id.org/credentials#",
+    "dc": "http://purl.org/dc/terms/",
+    "schema": "http://schema.org/",
+    "sec": "https://w3id.org/security#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "CryptographicKey": "sec:Key",
+    "GraphSignature2012": "sec:GraphSignature2012",
+    "Identity": "https://w3id.org/identity#Identity",
+    "LinkedDataSignature2015": "sec:LinkedDataSignature2015",
+    "created": {
+      "@id": "dc:created",
+      "@type": "xsd:dateTime"
+    },
+    "creator": {
+      "@id": "dc:creator",
+      "@type": "@id"
+    },
+    "domain": "sec:domain",
+    "expires": {
+      "@id": "sec:expiration",
+      "@type": "xsd:dateTime"
+    },
+    "nonce": "sec:nonce",
+    "normalizationAlgorithm": "sec:normalizationAlgorithm",
```

Comment:
I believe this has been renamed to `canonicalizationAlgorithm` (both terms appear in the security vocabulary but we decided to move forward with `canonicalizationAlgorithm`). Should I file a separate issue for this or should we just change it here?

Reply:
Let's make the change in this PR.

Reply:
👍